packages/next-plugin-sentry/src/on-init-client.js

```diff
@@ -1,10 +1,21 @@
-import * as Sentry from '@sentry/browser'
+import { init } from '@sentry/browser'
+import getConfig from 'next/config'
+
+import { getDsn, getRelease } from '../env'
+import { clientConfig } from '../config'
 
 export default async function initClient() {
-  // by default `@sentry/browser` is configured with defaultIntegrations
-  // which capture uncaughtExceptions and unhandledRejections
-  Sentry.init({
-    dsn: process.env.SENTRY_DSN,
-    release: process.env.SENTRY_RELEASE,
+  /**
+   * TODO(kamil): Unify SDK configuration options.
+   * RuntimeConfig cannot be used for callbacks and integrations as it supports only serializable data.
+   **/
+  const { publicRuntimeConfig = {} } = getConfig() || {}
+  const runtimeConfig = publicRuntimeConfig.sentry || {}
```

Comment:
`next/config` should not be used. This is not compatible with Next.js' SSG mode and only exists for legacy reasons.

Reply:
@Timer What do you suggest to replace it?